client/src/components/GasStation.jsx

```diff
@@ -29,15 +31,54 @@ const GasStationList = styled.ul`
   margin-bottom: 100px;
   flex-direction: column;
   gap: 10px;
+  width: 100%;
 `;
 
 const GasStationItem = styled.li`
   display: grid;
+  width: 100%;
+  grid-template-columns: 0.1fr 1fr 0.1fr 1fr 1fr 0.1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  gap: 0px 0px;
```

Comment:
I'm not sure if this is necessary. Otherwise, 0 is always 0. 0px is the same as 0rem, and so on. So you only need 0. :)